acmw-db-app/pages/api/meeting/[pid].js

```diff
@@ -0,0 +1,53 @@
+import pgQuery from '../../../postgres/pg-query.js';
+
+// GET /api/meeting/averageAttendance
+async function averageAttendance () {
+    const d = new Date();
```

Comment:
This is pretty redundant with the unique members per academic year. Maybe make a shared file that has commonly used functions like find current academic year?

Reply:
yeah I can do that for the ticket I'm working on rn (ADA-73)